src/components/sidebar.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { useMobileView } from "../context/mobileViewContext";
+
+function Sidebar() {
+  const { mobileViewActive, setMobileViewActive } = useMobileView();
+  return (
+    <div
+      className={
+        mobileViewActive === false
+          ? "vid-sidebar"
+          : "vid-sidebar sidebar-active "
+      }
```

Comment:
instead of doing this you can just do className={ ` vid-sidebar ${mobideView && "sidebar-active"} `}